tarpc/src/macros.rs

```diff
@@ -569,6 +581,20 @@ mod functional_test {
         assert_eq!(3, client2.add(1, 2).get().unwrap());
     }
 
+    #[test]
+    fn async_try_clone_unix() {
+        let handle = Server.spawn_with_config(UnixTransport("/tmp/test"),
```

Comment:
Use this: https://crates.io/crates/tempdir? I think that generates random directories, so you can do `Tempdir::new().unwrap().path().join("file");`